lib/natalie/compiler/backends/cpp_backend.rb

```diff
@@ -297,6 +305,30 @@ def init_symbols
         end
       end
 
+      def init_interned_strings
+        return [] if @interned_strings.empty?
+
+        # Start with setting the interned strings list all to nullptr and register the GC hook before creating strings
+        # Otherwise, we might start GC before we finished setting up this structure if the source contains enough strings
+        interned_strings = [
```

Comment:
I was confused for a sec because this `interned_strings` has the same name as the instance variable. Can we rename this to something like `lines` or `code` or whatever?